packages/fast-components-react-base/src/text-area/text-area.schema.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Text area",
+    "description": "A Text area component's schema definition.",
+    "type": "object",
+    "id": "text-area",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "readOnly": {
+            "title": "Read only",
```

Comment:
Read-only should be hyphenated. Also, may the title should be "HTML read-only attribute"? In text-field we do not make this a first-class prop... what is the reasoning of doing it here?

Reply:
This property was removed and needs to just be removed from the schema.

Reply:
removed